week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -17,37 +17,76 @@ Optional 2: Can you implement the shuffle functionality for your music player?
 This means the order the songs are played in will be random, but each song will only play once.
 
  */
-
-
+let i;
```

Comment:
this is not a very good idea, as you want any items in the class to be internal. 
JS is OOP language in concept, things like encapsulation is important, so you don't want (i) to get mixed with other classes